packages/cli/commands/project/logs.js

```diff
@@ -4,253 +4,154 @@ const {
+    getEnv(accountId) === 'qa' ? ENVIRONMENTS.QA : ENVIRONMENTS.PROD
   );
 
   return `${baseUrl}/private-apps/${accountId}`;
 };
 
-// We currently cannot fetch logs directly to the CLI. See internal CLI issue #413 for more information.
-
-// const handleLogsError = (e, name, projectName) => {
-//   if (e.response && e.response.status === 404) {
-//     logger.debug(`Log fetch error: ${e.message}`);
-//     logger.log(i18n(`${i18nKey}.logs.noLogsFound`, { name }));
-//   } else {
-//     logApiErrorInstance(
-//       e,
-//       new ApiErrorContext({ accountId: getAccountId(), projectName })
-//     );
-//     process.exit(EXIT_CODES.ERROR);
-//   }
-// };
-
-// const handleFunctionLog = async (accountId, options) => {
-//   const {
-//     latest,
-//     follow,
-//     compact,
-//     appPath,
-//     functionName,
-//     projectName,
-//   } = options;
-
-//   let logsResp;
+const handleFunctionLog = async (accountId, options) => {
+  const { functionName } = options;
 
-//   const tailCall = async after => {
-//     try {
-//       return appPath
-//         ? getProjectAppFunctionLogs(
-//             accountId,
-//             functionName,
-//             projectName,
-//             appPath,
-//             {
-//               after,
-//             }
-//           )
-//         : getFunctionLogs(accountId, functionName, { after });
-//     } catch (e) {
-//       handleLogsError(e, functionName, projectName);
-//     }
-//   };
+  // const { latest, follow, compact, functionName } = options;
+  // let logsResp;
 
-//   const fetchLatest = async () => {
-//     return appPath
-//       ? getLatestProjectAppFunctionLog(
-//           accountId,
-//           functionName,
-//           projectName,
-//           appPath
-//         )
-//       : getLatestFunctionLog(accountId, functionName, projectName);
-//   };
-
-//   if (follow) {
-//     await tailLogs({
-//       accountId,
-//       compact,
-//       tailCall,
-//       fetchLatest,
-//       name: functionName,
-//     });
-//   } else if (latest) {
-//     try {
-//       logsResp = await fetchLatest();
-//     } catch (e) {
-//       handleLogsError(e, functionName, projectName);
-//       return true;
-//     }
-//   } else {
-//     try {
-//       logsResp = await tailCall();
-//     } catch (e) {
-//       handleLogsError(e, functionName, projectName);
-//       return true;
-//     }
-//   }
+  // if (follow) {
+  //   await tailLogs({
+  //     accountId,
+  //     compact,
+  //     tailCall: ProjectLogsManager.tailCall,
+  //     fetchLatest: ProjectLogsManager.fetchLatest,
+  //     name: functionName,
+  //   });
+  // } else if (latest) {
+  //   logsResp = await ProjectLogsManager.fetchLatest();
+  // } else {
+  //   logsResp = await ProjectLogsManager.tailCall();
+  // }
+  //
+  // if (logsResp) {
+  //   outputLogs(logsResp, options);
+  // }
+  // return !!logsResp;
+  logger.log(i18n(`${i18nKey}.logs.noLogsFound`, { name: functionName }));
```

Comment:
Might make sense to mute this no logs found message for now.

Reply:
Sounds good, I'll delete it for now

Reply:
https://github.com/HubSpot/hubspot-cli/pull/1131/commits/f88732c070b9f991f8fbdd5ee8094ec353328126